linux/guest_customization/gosc_support_matrix.yml

```diff
@@ -101,3 +101,16 @@ OracleLinux:
       7.0.1: N/A
       7.0.2: N/A
       7.0.3: 20051473
+
+# Pardus 21.x / 23.x support GOSC testing on vCenter Server 8.0U2 and on vCenter Server 7.0.3 from build 22585855(7.0.3P08), or higher vCenter Server versions.
+Pardus GNU/Linux:
+  '21.0':
+    vcenter:
+      7.0.3: 22585855
+      8.0.0: N/A
+      8.0.1: N/A
```

Comment:
7.0.3 build should be 22357613, and you should add one more line of 8.0.2 GA build after 8.0.1.